projects/arcus.cmake

```diff
@@ -3,6 +3,7 @@ if(BUILD_OS_WINDOWS)
     set(extra_cmake_args
         -DCMAKE_LIBRARY_PATH=${CMAKE_INSTALL_PREFIX}/libs
         -DMSVC_STATIC_RUNTIME=ON
+        -DCMAKE_CXX_FLAGS="/std:c++17"
```

Comment:
A more portable way to implement this would be to adjust the CMake variable that was made for this:

https://cmake.org/cmake/help/latest/prop_tgt/CXX_STANDARD.html

You shouldn't need to make it platform-dependent then.